app/transport/rest/middleware.go

```diff
@@ -23,10 +23,10 @@ func WithLogRequest(logger *lgr.Logger) Middleware {
 	return func(h http.Handler) http.Handler {
 		return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
 			logger.Debugw("Request:",
-				"Method", req.Method,
-				"URL", req.URL,
-				"User-Agent", req.UserAgent(),
-			)
+				"Method",
```

Comment:
something is wrong with this formatting
1
2
1
2

keep pairs or a single var on a line
just keep a consistent approach

Reply:
I had some troubles with the linter here. Will fix it